corehq/ex-submodules/phonelog/tests/test_sumologic.py

```diff
@@ -18,15 +18,17 @@ def setUp(self):
         self.received_on = datetime.utcnow()
 
     def _get_xform(self, filename):
-        xform = FormProcessorInterface(self.domain).new_xform(convert_xform_to_json(self.get_xml(filename)))
+        xml = self.get_xml(filename)
+        xform = FormProcessorInterface(self.domain).new_xform(convert_xform_to_json(xml))
         xform.received_on = self.received_on
+        xform.get_xml = lambda: xml
```

Comment:
where is `xform.get_xml` used?

Reply:
It's called by `XFormInstance.form_data`, which is called by 

https://github.com/dimagi/commcare-hq/blob/36c17416b7beb15d762852de0eac51b8c6b00b74/corehq/form_processor/models/forms.py#L621

which is accessed by `SumoLogicLog.log_subreport`

https://github.com/dimagi/commcare-hq/blob/36c17416b7beb15d762852de0eac51b8c6b00b74/corehq/ex-submodules/phonelog/utils.py#L236

This patch was needed because the default implementation of `get_xml` hits the database, which is not allowed in a `SimpleTestCase`.